<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Depletion</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Depletion parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Output</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QGridLayout" name="EditorLayout"/>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="ExportData_PB">
             <property name="text">
              <string>Export data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ClearData_PB">
             <property name="text">
              <string>Clear data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Exit_PB">
             <property name="text">
              <string>Exit_PB</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="CreateDepletion_PB">
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Transport operator</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="2" column="1">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Fission-Q (in eV)</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QCheckBox" name="Prev_Res_CB">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Previous results</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QLineEdit" name="FissYieldOpts_LE">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QComboBox" name="FissYieldMode_CB">
                <item>
                 <property name="text">
                  <string>select mode</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>constant</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>cutoff</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>average</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_1">
                <property name="text">
                 <string>Transport Operator</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QComboBox" name="Diff_Vol_Meth_CB">
                <item>
                 <property name="text">
                  <string>select item</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>divide equally</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>match cell</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Chain file</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Fission yield mode</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QCheckBox" name="DiffBurnMats_CB">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>diff_burnable_mats</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Normalization mode</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Diff volume method</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QComboBox" name="Op_CB">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>CoupledOperator</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>IndependentOperator</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Reaction rate mode</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QComboBox" name="NormMod_CB">
                <item>
                 <property name="text">
                  <string>select item</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>energy-deposition</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>fission-q</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>source-rate</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="Chain_CB">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>chain_simple.xml</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>chain_endfb71_pwr.xml</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>chain_endfb71_sfr.xml</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>chain_endfb80_pwr.xml</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>chain_endfb80_sfr.xml</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>chain_casl_pwr.xml</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>chain_casl_sfr.xml</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="Fission_Q_LE">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Fission yield opts</string>
                </property>
               </widget>
              </item>
              <item row="10" column="0">
               <widget class="QComboBox" name="RRMode_CB">
                <item>
                 <property name="text">
                  <string>select mode</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>flux</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>direct</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>reaction_rate_opts</string>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <widget class="QLineEdit" name="RR_Opts_LE">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="10" column="2">
               <widget class="QSpinBox" name="RedChainLevel_SP">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="9" column="2">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Reduce chain level</string>
                </property>
               </widget>
              </item>
              <item row="8" column="2">
               <widget class="QCheckBox" name="ReduceChain_CB">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Reduce chain</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Integrator</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QComboBox" name="Integrator_CB">
              <item>
               <property name="text">
                <string>PredictorIntegrator</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CECMIntegrator</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CELIIntegrator</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CF4Integrator</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EPCRK4Integrator</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>LEQIIntegrator</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SICELIIntegrator</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SILEQIIntegrator</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Keff</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Nuclides</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Volume (in cc)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <layout class="QGridLayout" name="gridLayout_13">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="From_Nuclide_CB">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>152</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>from nuclides</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="Keff_LE">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="Nuclides_LE">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="Volume_LE">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="Modelcomponents">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Model components</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Geometry name</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Materials name</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Settings name</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="Geom_LE">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="Mats_LE">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="Sett_LE">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>166666</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="2">
               <widget class="QComboBox" name="TimeUnit_CB">
                <item>
                 <property name="text">
                  <string>d</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>h</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>min</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>s</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLineEdit" name="Power_LE">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_4">
                <property name="minimumSize">
                 <size>
                  <width>68</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Time units</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Power</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Time steps</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QLineEdit" name="TimeSteps_LE">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>220</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QComboBox" name="Power_Unit_CB">
                <item>
                 <property name="text">
                  <string>W</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>KW</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>MW</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Power units</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Geom_LE</tabstop>
  <tabstop>Sett_LE</tabstop>
  <tabstop>CreateDepletion_PB</tabstop>
  <tabstop>ExportData_PB</tabstop>
  <tabstop>ClearData_PB</tabstop>
  <tabstop>Exit_PB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
